report/src/introduction/waterfall_vs_agile.tex

```diff
@@ -1,2 +1,60 @@
+During collaboration with the stakeholder, it is expected that requirements will be explored incrementally, and that they will provide feedback periodically. This places the project on the agile end of the dynamism dimension.
+Finally, the personnel \& culture dimensions also lead toward the agile end for this project.\cite{boehmObservationsBalancingDiscipline}
+
+Based on this analysis, it is concluded that an agile approach is an appropriate process model for this project.
+
+Agile is a set of values, and does not present practical activities which a process model would consist of. Therefore, the team has decided on Scrum, which is an agile framework that puts the values to practice. This decision was made due to prior experience with using the framework amongst team members.
+
+Scrum introduces a variety  of new practices and roles. In a Scrum team, there is a Scrum Master, a Product Owner, and developers. The Scrum Master is responsible for enforcing scrum and helping the team remain productive. Their goal is to guide the team towards continuous improvement. A Product Owner is responsible for maximizing value from the product. They are also accountable for managing the product backlog, which includes developing a product goal, creating backlog items and prioritizing them, and making sure the team understands this backlog.
+
+The process for working as a Scrum team boils down to repeatedly taking backlog items and working on them during a sprint. After the sprint, the team and stakeholders inspect the results and adjust for the next sprint. This process is what fosters continuous improvement by getting rapid feedback on work.
+
+During this report, we will present the work done during sprints. This includes the initial planning, and later the product feedback received.
+To find common understanding of the process, we present definitions of the artifacts and tools used during the Scrum process below.
+
+One of the first things a Scrum team implements is the \textbf{product backlog}. This is a list of user stories, and represents every possible addition to the project. Due to the comprehensive nature of this list, it is not possible to build everything. Therefore, the list should be prioritized by what delivers the most value to users.
+Backlog items then are estimated by the team in terms of relative size. A common measure for estimation is the Fibonacci sequence.
+As the stakeholders provide feedback, items in the backlog should be prioritized accordingly. All these items contribute to achieving the \textbf{product goal}.
+
+The backlog items are \textbf{user stories}. These stories represent functionality the stakeholder would like to see in the system. For this reason, they must include \textit{who} it is for, \textit{what} should be done, and \textit{why} they want it. This is captured in the general "\texttt{As a X, I want Y, such that Z.}" format for user stories.
```

Comment:
```suggestion
User stories represent functionality the stakeholder would like to see in the system. For this reason, they must include \textit{who} it is for, \textit{what} should be done, and \textit{why} they want it. This is captured in the general "\texttt{As a X, I want Y, such that Z.}" format for user stories.
```

Reply:
Hmm. I think specifying that the backlog items are user stories is better.